content/en/blog/_posts/2021-02-27-annotating-k8s-for-humans.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog
+title: 'Annotating Kubernetes Services for Humans'
+date: 2021-02-27
+slug: annotating-k8s-for-humans
+---
+
+**Authors:** Richard Li (Ambassador Labs)
+
+Have you ever been asked to troubleshoot a failing Kubernetes service and struggled to find basic information about the service such as the source repository and owner?
+
+One of the problems as Kubernetes applications grow is the proliferation of services. As the number of services grows, developers start to specialize working with specific services. When it comes to troubleshooting, however, developers need to be able to find the source, understand the service and dependencies, and chat with the owning team for any service.
+
+## Human Service Discovery
```

Comment:
Our style guide prefers headings at HTML `<h2>` and higher to be in sentence case, so:
```suggestion
## Human service discovery
```